extension/manifest.json

```diff
@@ -10,7 +10,8 @@
 	},
 	"permissions": [
 		"alarms",
-		"activeTab"
+		"activeTab",
+		"tabs"
```

Comment:
Yes, If I don't check for undefined, it works without the tabs permission. Seems to me a bug in the chrome security model? How can I query all tabs for urls without the tabs permission? :/